tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -108,7 +108,6 @@ def get_regex_buckets(event):
     """
     buckets = {}
     try:
-        level = "event['config']"
         buckets["protected"] = event["config"]["protected-bucket"]
```

Comment:
If you recall from our conversation yesterday, I would cleanup, fix this part of the code with the buckets. Currently, the code expects a bucket with a key with a value of one of (public, protected, internal, or private). This will break if a user configures their Cumulus with a bucket named something like public_2 which is valid in Cumulus. I would change this to get the full bucket configuration mapping so that we could search for the correct key and get the name of the bucket. This would also require a change to the step function JSON as well to pull all the buckets in the configuration section.

Reply:
I am still having some confusion with how it will work.  The bucket names are say for example, 
`podaac-sndbx-cumulus-internal`,  `podaac-sndbx-cumulus-private`, `podaac-sndbx-cumulus-protected`, `podaac-sndbx-cumulus-public`

why will a user name it `public_2`?

Reply:
also for `modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json`, should this be changed?

```          "task_config": {
            "glacier-bucket": "${orca_default_bucket}",
            "public-bucket": "{$.meta.buckets.public.name}",
            "protected-bucket": "{$.meta.buckets.protected.name}",
            "private-bucket": "{$.meta.buckets.private.name}",
            "internal-bucket": "{$.meta.buckets.internal.name}",
            "file-buckets": "{$.meta.collection.files}",
            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}"
          }
````

Reply:
Per our conversation this morning, yes, modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json will need to be changed as well as the code. 

`public_2` refers to the key name from the buckets configuration not the actual bucket name itself.